modules/administration/pages/maintenance-window.adoc

```diff
@@ -1,36 +1,45 @@
 [[maintenance-window]]
-= Maintenance Window Checklist
+= Maintenance Window
 
-If you work on a {productname} Server with scheduled maintenance windows, you might find it difficult to remember all the things that you need to during that critical downtime.
+If you work with scheduled maintenance windows, you might find it difficult to remember all the things that you need to do before, during, and after that critical downtime of the {productname} Server.
+{productname} Server related systems such as Inter-Server Synchronization Slave Servers or {productname} Proxies are also affected and have to be considered.
 
-As with any {sle} system, you are encouraged to install package updates on the {susemgr} Server when they arrive.
+{suse} recommends you always keep your {productname} infrastructure updated.
+That includes servers, proxies, and build hosts.
```

Comment:
Thinking about this: if we mention build host, I wonder if we shouldn't mention monitoring host (the clients with grafana and prometheus). @cavalheiro @renner?

Reply:
As of now, we still do not have provisioning of Prometheus and Grafana servers. But its worth mentioning once we have it.

Reply:
Not automated, but users can still install grafana and prometheus on clients following manual instructions from the doc, right?

If so, they should update such clients with grafana and prometheus together with the server, the proxy and the build hosts.

Reply:
That's right, I agree.